projects/unions-and-literals/narrow-trails/src/index.test.ts

```diff
@@ -33,5 +38,6 @@ describe(runCommands, () => {
 		const actual = runCommands();
 
 		expect(actual).toEqual(expected);
+		expect(mockRandom).toHaveBeenCalledTimes(randomValues.length);
```

Comment:
Just to make sure there aren't any shenanigans happening with random numbers between tests...